src/composition/supergraph/config/resolver/mod.rs

```diff
@@ -144,6 +144,12 @@ pub enum LoadSupergraphConfigError {
     /// IO error that occurs when the supergraph contents can't be access via File IO or Stdin
     #[error("Failed to read file descriptor. Error: {0}")]
     ReadFileDescriptor(RoverError),
+    /// Occurs when a supergraph cannot be deserialised, ready for expansion
+    #[error("Failed to deserialise the supergraph config. Error: {0}")]
+    DeserialisationError(#[from] Error),
```

Comment:
__NIT__: American spelling and use full name to avoid confusion

```suggestion
    DeserializationError(#[from] serde_yaml::Error),
```